ONSdigital/ssdc-rm-support-tool
# Upgrade prod node packages

# Motivation and Context
Upgrade to prod node packages to please the npm audit

# What has changed
Upgrade to prod node packages.  Div tags no longer allow align attributes, so changed those as well.

For reasons here, the dev packages don't have any effect on the app and false positives are shown in the dev audit (https://github.com/facebook/create-react-app/issues/11174), so they can be ignored.

A few changes to keep Travis happy: 
- upgraded node to version 16, as this is what the Concourse pipeline uses
- upgraded Prettier to 3.0.0 because this is the version that Travis uses.
- lots of formatting changes to keep Travis happy, but it's all minor stuff

# How to test?
Run `make package-audit`.  Everything should pass.
Build the image by running `make build` then run it - it should build and everything in the UI should look normal.

# Links
https://trello.com/c/lfFg5dJd
